src/TeamHydrator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace TheRealMVP;
+
+class TeamHydrator 
+{
+    /**
+     * create database connection and retrieves data and returns an array of team objects
+     *
+     * @return array
+     */
+    public static function getData() : array
+    {
+        $pdo = new \PDO('mysql:host=db; dbname=TheRealMVP', 'root', 'password');
```

Comment:
I don't like how you create two PDO objects, would suggest adding a PDO property to this class, like you have in the importer class, and then pass in a PDO connection from outside via the constructor, this means that you can recycle a single connection.